omnibus/package-scripts/agent/prerm

```diff
@@ -161,8 +161,8 @@ else
 fi
 
 # Delete all .pyc files in the `agent/` and the `bin/agent/dist` dirs
-find $INSTALL_DIR/agent -name '*.py[co]' -type f -delete || echo "Unable to delete .pyc files in $INSTALL_DIR/agent"
-find $INSTALL_DIR/bin/agent/dist -name '*.py[co]' -type f -delete || echo "Unable to delete .pyc files in $INSTALL_DIR/bin/agent/dist"
-find $INSTALL_DIR/bin/agent/dist -name '__pycache__' -type d -delete || echo "Unable to delete __pycache__ directories in $INSTALL_DIR/bin/agent/dist"
+find $INSTALL_DIR/embedded/lib/python* -name '*.py[co]' -type f -delete || true
```

Comment:
the `embedded` folder can contain user-installed packages that only ship `*.py[co]` files, and we want to preserve these, so we shouldn't remove all these files from the embedded directory. I think we should remove this line.

Also, note that the removal of pyc/pyo files from the embedded directory is handled separately in the package scripts, based on a file listing all the known pyc/pyo files.

(in v5, `$INSTALL_DIR/agent` only contained the agent source code, not its python deps, so it was fine to remove the pyo/pyc files there)